### ISSUE TITLE
Allow reorder of previous & next buttons in <amp-carousel>

### ISSUE THREAD

**Issue by cbecker87** [ampproject/amphtml#30517]
Currently, when using <amp-carousel>, the previous and next buttons are automatically created and placed in the HTML after the carousel content.  When using amp-carousel with the loop attribute set and the type set to "slides", if the images within the carousel contain links or other focusable content, the tab order for keyboard accessiblity is incorrect.  There should be an option to dictate where the previous and next buttons are placed within the HTML - in this case, the previous button should be located prior to the carousel content, and the next button should be located after the carousel content.  Then, as a user tabs through a page, they will encounter the previous button first, then the content within the current slide (as all 'hidden' slides have the aria-hidden attribute set to true), and then the next button.  This allows keyboard accessibility to function as expected for keyboard users.

**Comment by caroqliu** [comment]
We unfortunately cannot reorder the buttons in the DOM as this could be a breaking change if anyone is relying on `:nth-child` CSS selectors. Another option could be to give an increasing `tabindex` between the previous button, slides, and next buttons, but seeing as we cannot control or coordinate the user provided `tabindex` on slides, this may not be fixable without breaking the current versions.

However this is something that is fixed in the new Bento versions of the carousel, see #28284.
